### PR TITLE
Graph overlay was expanding off the screen to the size of the prompt line

### PR DESCRIPTION
**Pull request by mickr777**
sure this isn't really important at the moment

just limited the size of the width and gave it a shadow
image

### CODE REVIEW

**Comment by blessedcoolant** [comment]
The Node Editor is very WIP at the moment but no worries at all. If something is bothering you, feel free to pitch up a fix. It all adds up for later anyway. Thanks for the PR.

File: invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx [hunk 1]
```diff
@@ -13,14 +13,16 @@ const NodeGraphOverlay = () => {
...
+      top={2}
       right={2}
       opacity={0.7}
       background="base.800"
       p={2}
       maxHeight={500}
+      maxWidth={500}
       overflowY="scroll"
       borderRadius="md"
+      boxShadow="0px 10px 10px rgba(0, 0, 0, 0.25)"
```
Comment (blessedcoolant) [inline invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx:25]: `boxShadow="0px 10px 10px rgba(0, 0, 0, 0.25)"`

You want to use the colors from the existing theme palette so it translates correctly to all themes. But I need to check if we have it setup for transparency. If not, we check if there's a `shadowOpacity` parameter in chakra to induce it.

**Review by blessedcoolant — CHANGES_REQUESTED** [review]

File: invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx (hunk 1 above)
Comment (mickr777) [reply invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx]: I couldn't find the one for shadows, so I will just remove the shadow

**Review by blessedcoolant — APPROVED** [review]

File: invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx (hunk 1 above)
Comment (psychedelicious) [reply invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx]: @mickr777 for future reference, [chakra has some built-in shadows](https://chakra-ui.com/docs/styled-system/style-props#shadow)

if you want to add a shadow, you can do it in `invokeai/frontend/web/src/theme/theme.ts`

one thing to note is you cannot use the theme tokens in the shadow string; you have to use CSS variables, as demonstrated in `theme.ts`

then to use the shadow, you'd do `boxShadow='dark.accent'` or `boxShadow='light.accentHover'`.

we aren't using any of these custom shadows anywhere; i usually use the built-in `dark-lg` shadow

Comment (mickr777) [reply invokeai/frontend/web/src/features/nodes/components/NodeGraphOverlay.tsx]: @psychedelicious thank you for the info

**Pull request merged**